apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
+// eslint-disable-next-line @typescript-eslint/typedef
+const importLazy = require('import-lazy')(require);
```

Comment:
The `importLazy()` pattern looks like something we should roll out across all our projects.

Should we bake it into `node-core-library`?  Then we could eliminate the extra **package.json** dependency and make it more concise, something like this:

```
import { Import } from '@rushstack/node-core-library';

// eslint-disable-next-line @typescript-eslint/typedef
const fsx = Import.requireLazy('fs-extra', require);
```

We could also wrap [resolve](https://www.npmjs.com/package/resolve) behind this API as `Import.resolve()`, since there's some stuff I would like to improve/validate about how we call `resolve()` in our code.

Reply:
Could we also look at using `import type` to preserve typing and `downlevel-dts` to prevent breaking consumers who use older versions of TypeScript?

Reply:
Yes `downlevel-dts` is getting to be a high priority. I don't think we can use it directly though; we may have to make a Rush Stack fork that is more professional. I believe Fabric came to the same conclusion when they tried to incorporate it into their toolchain. It is missing basic stuff like an API and ability to operate on strings in memory. I keep hoping Microsoft will take ownership of `downlevel-dts` and make it a supported feature of the compiler.

Reply:
I have changed all Import.lazy calls to also import types. I will resolve this comment for now. Feel free to reopen though. :-)